src/MemberBundleProblemType.ts

```diff
@@ -0,0 +1,8 @@
+export enum MemberBundleProblemType {
+  MEMBER_NOT_FOUND = 'https://veraid.net/problems/member-not-found',
+  MEMBER_NO_NAME = 'https://veraid.net/problems/member-no-name',
+  ORG_NOT_FOUND = 'https://veraid.net/problems/org-not-found',
+  PUBLIC_KEY_NOT_FOUND = 'https://veraid.net/problems/member-public-key-not-found',
+  BUNDLE_GENERATION_ERROR = 'https://veraid.net/problems/member-bundle-generation-error',
+
+}
```

Comment:
We could import the errors from the other problem types like:
```
MEMBER_NOT_FOUND = MemberProblemType.MEMBER_NOT_FOUND
```
or we could chenge the method return type:
```
Promise<Result<ArrayBuffer, MemberBundleProblemType>>
```
to
```
Promise<Result<ArrayBuffer, MemberBundleProblemType | MemberProblemType | OrgProblemType>>

```